packages/api/src/models/cropManagementPlanModel.js

```diff
@@ -86,6 +86,34 @@ class CropManagementPlanModel extends Model {
       },
     };
   }
+
+  // Custom function used in copy crop plan
+  // Should contain all jsonSchema() and relationMappings() keys
+  static get templateMappingSchema() {
+    return {
+      // jsonSchema()
+      management_plan_id: 'omit',
+      seed_date: 'edit',
+      plant_date: 'edit',
+      germination_date: 'edit',
+      transplant_date: 'edit',
+      harvest_date: 'edit',
+      termination_date: 'edit',
+      //TODO: convert already in ground to edit when planting method is known
```

Comment:
This `TODO` is valid? If doesn't need to addressed, let's remove it or create a different ticket to tackle it later